src/xsl/openrosa2html5form.xsl

```diff
@@ -500,11 +508,11 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
                         </xsl:variable>
                         <xsl:element name="{$element}">
                             <xsl:choose>
-                                <xsl:when test="not(local-name() = 'setvalue') and ancestor::odk:rank">
+                                <xsl:when test="not(local-name() = 'setvalue' or local-name() = 'setgeopoint') and ancestor::odk:rank">
                                     <xsl:call-template name="rank-item-attributes"/>
                                 </xsl:when>
-                                <xsl:when test="local-name() = 'setvalue'">
-                                    <xsl:call-template name="setvalue-attributes">
+                                <xsl:when test="local-name() = 'setvalue' or local-name() = 'setgeopoint'">
+                                    <xsl:call-template name="setvalue-setgeopoint-attributes">
```

Comment:
A more general name might be `nested-action-attributes`? Not clear it's better now that there are only two relevant actions.

Reply:
That does seem like a better name. Wonder what @MartijnR thinks.

Reply:
I like `action-attributes`. And generalizing on the term "action" has felt pretty good as I've used it in enketo-core.